gatk-launch

```diff
@@ -145,13 +156,17 @@ def getSparkSubmitCommand(sparkSubmitCommand):
 def getLocalGatkRunCommand():
     localJarFromEnv = getJarFromEnv(GATK_LOCAL_JAR_ENV_VARIABLE)
     if localJarFromEnv is not None:
-        return ["java", "-jar", localJarFromEnv]
+        return formatLocalJarCommand(localJarFromEnv)
 
     wrapperScript = getGatkWrapperScript(throwIfNotFound=False)
     if wrapperScript is not None:
         return [wrapperScript]
 
-    return ["java", "-jar", getLocalJar()]  # will throw if local jar not found
+    return formatLocalJarCommand(getLocalJar())  # will throw if local jar not found
+
+
+def formatLocalJarCommand(localJarFromEnv):
+    return ["java"] + PACKAGED_LOCAL_JAR_OPTIONS  + [ "-jar", localJarFromEnv]
```

Comment:
`localJarFromEnv` should be renamed, since it's not necessarily from the environment. Just `localJar` is probably fine.